Brisk/Extensions/NSTextView+Extension.swift

```diff
@@ -2,12 +2,12 @@ import AppKit
 
 extension NSTextView {
     var stringValue: String {
-        return self.string ?? ""
+        return self.string
     }
 }
 
 extension NSTextView: Validatable {
     var isValid: Bool {
-        return self.string?.isEmpty == false
+        return self.string.isEmpty == false
```

Comment:
We can use ! Here instead of == false now

Reply:
👍